### PR TITLE
14392 created symbolic links for completeness test

### PR DESCRIPTION
**Pull request by saricicekibrahim**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -3,3 +3,6 @@
 set -e
 
 sudo apt-get install -y curl awscli
+
+ln -s ${GEOCINT_WORK_DIRECTORY}/completeness_report/ ${GEOCINT_WORK_DIRECTORY}/public_html/completeness_report/
```
Comment (Zkir) [inline install.sh:7]: it's install.sh, and it installs dependancies.
is `${GEOCINT_WORK_DIRECTORY} /geocint` created already? or it is created when geocint pipeline is started first time?

Also, it seems that `${GEOCINT_WORK_DIRECTORY}/completeness_report/ `shoud be rather `${GEOCINT_WORK_DIRECTORY}/**geocint**/completeness_report/`

**Review by frolui — APPROVED** [review]

**Review by Zkir — CHANGES_REQUESTED** [review]

File: install.sh (hunk 1 above)
Comment (Zkir) [inline install.sh:7]: Also, it seems that `${GEOCINT_WORK_DIRECTORY}/completeness_report/ `shoud be rather `${GEOCINT_WORK_DIRECTORY}/**geocint**/completeness_report/`

Comment (saricicekibrahim) [reply install.sh:7]: on Stratum both works
https://stratum.kontur.io/geocint/report/completeness.html
https://stratum.kontur.io/report/completeness.html
I think no need to use /geocint?

Comment (saricicekibrahim) [reply install.sh:7]: changed folder structure

File: install.sh [hunk 2]
```diff
@@ -3,3 +3,6 @@
...
+ln -s ${GEOCINT_WORK_DIRECTORY}/report/ ${GEOCINT_WORK_DIRECTORY}/public_html/report/
```
[the remark this answers is not in the document]
Comment (saricicekibrahim) [reply install.sh:8]: changed and tested together

File: install.sh (hunk 1 above)
Comment (saricicekibrahim) [reply install.sh]: no need to have folders to create symbolic link. tested on stratum

**Review by Zkir — APPROVED** [review]

**Pull request merged**